app/main.py

```diff
@@ -1,2 +1,83 @@
+from dataclasses import dataclass
+from typing import Any, Hashable
+
+
+@dataclass
+class Node:
+    key: Any
+    value: Any
+    hash_: Hashable
```

Comment:
What type does hash function return?
Also consider to rename this variable, to make it more descriptive